src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/payment/FeesAndStatusCheckPreparer.java

```diff
@@ -96,65 +97,58 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
-                            && (callback.getEvent() == Event.PAYMENT_APPEAL)
-                        ) {
-                            asylumCasePreSubmitCallbackResponse.addError(oldOrExistingCasesLabel);
-
+                        if (paymentStatus.equals(PAID)
+                            || remissionPartiallyApproved
+                            || remissionApproved) {
+                            asylumCasePreSubmitCallbackResponse.addError(PAYMENT_OPTION_NOT_AVAILABLE_LABEL);
                         }
-
-                        eaHuPaymentType
-                            .filter(option -> option.equals("payOffline")
-                                              || paymentStatus == PaymentStatus.PAID)
-                            .ifPresent(s -> {
-                                if (callback.getEvent() == Event.PAYMENT_APPEAL) {
-
-                                    asylumCasePreSubmitCallbackResponse.addError(paymentOptionNotAvailableLabel);
-                                }
-                            });
                         break;
 
                     case PA:
-                        Optional<String> paPaymentType = asylumCase.read(PA_APPEAL_TYPE_PAYMENT_OPTION, String.class);
+                        String paPaymentType = asylumCase.read(PA_APPEAL_TYPE_PAYMENT_OPTION, String.class).orElse("");
+                        String paAipPaymentType = asylumCase.read(PA_APPEAL_TYPE_AIP_PAYMENT_OPTION, String.class).orElse("");
+
                         if (paPaymentType.isEmpty()
-                            && (remissionDecision.isEmpty() || remissionDecision.get() != RemissionDecision.REJECTED)
-                            && (callback.getEvent() == Event.PAYMENT_APPEAL)
+                            && paAipPaymentType.isEmpty()
+                            && (remissionDecision.isEmpty() || remissionDecision.get() != REJECTED)
+                            && paymentAppealEvent
                         ) {
-                            asylumCasePreSubmitCallbackResponse.addError(oldOrExistingCasesLabel);
-
+                            asylumCasePreSubmitCallbackResponse.addError(OLD_OR_EXISTING_CASES_LABEL);
                         }
 
-                        paPaymentType
-                            .filter(option -> option.equals("payOffline")
-                                              || paymentStatus == PaymentStatus.PAID
-                                              || (option.equals("payLater") && currentState == State.APPEAL_STARTED))
-                            .ifPresent(s -> {
-                                if (callback.getEvent() == Event.PAYMENT_APPEAL) {
+                        boolean paAppealStartedPayLater = (paPaymentType.equals("payLater") || paAipPaymentType.equals("payLater")) && currentState == APPEAL_STARTED;
+
+                        if ((isPaid || remissionPartiallyApproved || remissionApproved || paAppealStartedPayLater)
+                            && paymentAppealEvent) {
```

Comment:
To prevent payment event when PA appeal is paid, or has fully/partially approved remissions, or is in `appealStarted` state